current/runtime/events.md

```diff
@@ -87,7 +86,7 @@ decl_module! {
 
 The default behavior of this function is to call
 [`deposit_event`](https://substrate.dev/rustdocs/master/frame_system/struct.Module.html#method.deposit_event)
-from the SRML System module.
+from the System module, which writes the event to storage.
```

Comment:
```suggestion
from the FRAME system, which writes the event to storage.
```

I don't feel strongly about this one. Just thought it might be nice to get away from the word module.